filament/include/filament/Scene.h

```diff
@@ -111,6 +111,14 @@ class UTILS_PUBLIC Scene : public FilamentAPI {
      */
     void remove(utils::Entity entity);
 
+    /**
+     * Removes a contiguous list of entities to the Scene.
```

Comment:
I'm not sure "contiguous" is needed here -- when I read it the first time I thought the Entities needed to be contiguous.

I think we should state that this is equivalent to calling remove() on a single entity in a loop.

This is to address the case of the array containing the same Entity several times -- the effect will be "as if" it was removed several times.

Maybe specify what happens when entities is nullptr (e.g. it's forbidden, or it's a noop). I would go with noop (to mimic free()), but don't have a strong opinion.

Reply:
All good suggestions, I updated the commit with new comments.  (If any entity, including the null entity, does not exist in the scene, then the robin_set will silently ignore it.)